docs/src/examples/Optimal Control/quadcopter.jl

```diff
@@ -0,0 +1,140 @@
+# # Quadcopter Trajectory and Set Point Tracking
+
+# The quadcopter is an unmanned aerial vehicle with 4 propellors and in this case study,
+# we seek to determine an optimal control policy for the 
+# 4 control inputs that must be adjusted over a specified time period to enable the 
+# quadcopter to closely follow given set points/trajectory while minimizing the difference between 
+# the set point and actual position and propellor input
+
+# ## Background
+
+# Modelling the quadcopter trajectory is an infinite dimensional optimization problem.
+# The variables are dependent on time. 
+# Using a sinusoidal setpoint trajectory and 5 minute time horizon, the problem formulation is as follows:
+
+# ## Formulation
+
+# ```math 
+#\begin{aligned}
+#	&&\underset{x(t), u(t)}{\text{min}} &&& \int_{0}^{T} \left( (x_1 - \bar{x}_1)^2 + (x_3 - \bar{x}_3)^2 + (x_5 - \bar{x}_5)^2 + x_7^2 + x_8^2 + x_9^2 + 0.1 \sum_{i=1}^{4} u_i^2 \right) dt  \\
+#	&&\text{s.t.} &&& x_j(0) = 0, \quad j \in \{1,2,3,\dots,9\} \\
+#	&&&&& \frac{\partial x_j}{\partial t} = x_{j+1}, \quad j \in \{1,3,5\} \\
+#	&&&&& \frac{\partial x_2}{\partial t} = u_1 \cos(x_7) \sin(x_8) \cos(x_9) + u_1 \sin(x_7) \sin(x_9) \\
+#	&&&&& \frac{\partial x_4}{\partial t} = u_1 \cos(x_7) \sin(x_8) \cos(x_9) - u_1 \sin(x_7) \sin(x_9) \\
+#	&&&&& \frac{\partial x_6}{\partial t} = u_1 \cos(x_7) \cos(x_8) - 9.8 \\
+#	&&&&& \frac{\partial x_7}{\partial t} = u_2 \frac{\cos(x_7)}{\cos(x_8)} + u_3 \frac{\sin(x_7)}{\cos(x_8)} \\
+#	&&&&& \frac{\partial x_8}{\partial t} = - u_2 \sin(x_7) + u_3 \cos(x_7) \\
+#	&&&&& \frac{\partial x_9}{\partial t} = u_2 \cos(x_7) \tan(x_8) + u_3 \sin(x_7) \tan(x_8) + u_4 \\
+#	&&&&& \bar{x}_1 = \sin\left(\frac{2\pi t}{T}\right), \quad \bar{x}_3 = \sin\left(\frac{2\pi t}{T}\right), \quad \text{and} \quad \bar{x}_5 = \frac{2t}{T}
+#\end{aligned}
+# ```
+
+# ## Model Definition
+
```

Comment:
In all the sections below please reduce the number of spaces used. We shouldn't each line be orphaned by itself. Please refer to the other examples as a reference.